packages/web-app-files/tests/unit/components/SideBar/Links/PrivateLinkItem.spec.js

```diff
@@ -0,0 +1,116 @@
+import Vuex from 'vuex'
+import GetTextPlugin from 'vue-gettext'
+import DesignSystem from 'owncloud-design-system'
+import { shallowMount, createLocalVue } from '@vue/test-utils'
+import PrivateLinkItem from '@files/src/components/SideBar/Links/PrivateLinkItem.vue'
+
+const localVue = createLocalVue()
+
+localVue.use(Vuex)
+localVue.use(DesignSystem)
+localVue.use(GetTextPlugin, {
+  translations: 'does-not-matter.json',
+  silent: true
+})
+
+const selectors = {
+  linkItem: '.oc-files-private-link-item',
+  copyToClipboardButton: '.oc-files-private-link-copy-url'
+}
+
+global.window = Object.create(window)
+const windowLocation = { href: 'http://localhost:9100' }
+Object.defineProperty(window, 'location', {
+  value: windowLocation
+})
+
+describe('PrivateLinkItem', () => {
+  function createStore({ isMounted = true } = {}) {
+    return new Vuex.Store({
+      getters: {
+        capabilities: function() {
+          return {
+            files: {
+              privateLinks: true
+            }
+          }
+        }
+      },
+      modules: {
+        Files: {
+          namespaced: true,
+          getters: {
+            highlightedFile: function() {
+              return {
+                isMounted: jest.fn(() => isMounted),
+                name: 'lorem.txt',
+                privateLink: 'some-link'
+              }
+            }
+          }
+        }
+      }
+    })
+  }
+
+  function getShallowWrapper(store) {
+    return shallowMount(PrivateLinkItem, {
+      localVue,
+      store,
+      stubs: {
+        'copy-to-clipboard-button': true
+      }
+    })
+  }
+
+  describe('when private link is enabled', () => {
+    it('should be visible', () => {
+      const store = createStore()
+      const wrapper = getShallowWrapper(store)
+
+      expect(wrapper.find(selectors.linkItem).exists()).toBeTruthy()
+    })
+
+    it('should render copy to clipboard button', () => {
```

Comment:
```suggestion
    it('should render the copy to clipboard button', () => {
```

Reply:
```suggestion
    it('should render the copy-to-clipboard button', () => {
```
Are we going to "name" buttons by joining the words with `-`?